setup.py

```diff
@@ -19,6 +19,7 @@ def get_long_desc():
     'pytest',
     'pytest-cov',
     'mock',
+    'pre-commit',
```

Comment:
I don't think that this is something we should add to setup.py for this project. pre-commit is something the developer has to have installed on their machine globally, it is NOT a dependency of this project, not even a dev dependency IMHO.